Packs/CortexXDR/.pack-ignore

```diff
@@ -156,4 +156,7 @@ ignore=IF100
 ignore=PB105
 
 [file:playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR.yml]
-ignore=PB105
\ No newline at end of file
+ignore=PB105
+
+[file:Cortex_XDR_-_Large_Upload.yml]
+ignore=PB121
```

Comment:
for which value did you need to add this ignore? Cause now this might a problem for future errors with the same code ( it will be ignored automatically).

Reply:
I added this ignore since 'DBotFindSimilarIncidents' automation used in task number 4 triggers errors because the inputs are in simple values.